applications/FluidDynamicsApplication/tests/adjoint_fluid_test.py

```diff
@@ -1,39 +1,16 @@
 import KratosMultiphysics as km
-import KratosMultiphysics.FluidDynamicsApplication as kfd
-have_required_applications = True
-missing_applications_message = ["Missing required application(s):",]
-try:
-    import KratosMultiphysics.ExternalSolversApplication
```

Comment:
this is not used in this test

Reply:
I believe it used to be needed in order to import the solver factory, but this is no longer the case thanks to your new python import system. I suspect I can find other examples of this in the fluid application tests, will check later.

Reply:
I checked this test, it only uses amgcl, nothing from externalsolvers

Reply:
I know, but the linear solver factory used to contain a nice `from ExternalSolversApplication import *` that made it impossible to run the test without.

Reply:
ah ok, this has been removed, so now it should be fine